src/app/api/auth/[...nextauth]/options.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextAuthOptions } from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+
+export const options: NextAuthOptions = {
+  providers: [
+    GoogleProvider({
+      clientId: process.env.GOOGLE_CLIENT_ID!,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET!,
+    }),
+  ],
+  pages: {
+    signIn: "/login",
+  },
+  callbacks: {
+    async redirect({ baseUrl }) {
+      // Redirect users to the /profile route after login
```

Comment:
Current rerouting is done to base directory not profile

Reply:
Oops ya, forgot to change the comment but I decided to route to / instead of /profile because in the event that user log out, the same callback url will be used and it will be weird for users to be routed to /profile when they are not authenticated.